CHANGELOG.md

```diff
@@ -32,7 +32,8 @@
 
 ### Breaking Changes
 
-- TODO ([#XXX](https://github.com/kedacore/keda/pull/XXX))
+- **RabbitMQ Scaler:** Include `vhost` for RabbitMQ when retrieving queue info with `useRegex` ([#2591](https://github.com/kedacore/keda/pull/2591))
```

Comment:
Is this a breaking change? I mean, is not the same behavior as now if I keep `vHost` empty?

Reply:
Yeah, put it into improvements pls.

Reply:
Done